sport/app/football/datetime/dateHelpers.scala

```diff
@@ -33,8 +33,9 @@ object DateHelpers {
     ld.atStartOfDay(defaultFootballZoneId)
   }
 
+  // Note format of 'yyyyMMdd' required!
   def parseLocalDate(year: String, month: String, day: String): java.time.LocalDate = {
     val formatter = DateTimeFormatter.ofPattern("yyyyMMdd").withZone(DateHelpers.defaultFootballZoneId)
-    java.time.LocalDate.parse(s"$year$month$day", formatter)
+    java.time.LocalDate.parse(s"$year${month.capitalize}$day", formatter)
```

Comment:
This part is not actually related to the issue (and we haven't seen issues around this), but feels like a sensible precaution as there are parts where months are capitalised in the codebase and others where they are not.